contrib/easyconfig-templates/2022a/PythonBundle-scipy.eb.tmpl

```diff
@@ -0,0 +1,39 @@
+# Template for a python bundle using SciPy-bundle on 2022a
+easyblock = 'PythonBundle'
+
+name = 'SoftwareName'
+version = '0.0.0'
+
+homepage = 'https://www.domain.org'
+description = """
+Description in 80 chars long column
+"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('SciPy-bundle', '2022.05'),
+]
+
+use_pip = True
```

Comment:
@lexming This is enabled by default in EasyBuild 5.0.0, so this line should be removed

Reply:
Fixed in https://github.com/easybuilders/easybuild-easyconfigs/pull/21984/commits/40cff25e1d5e98997b285b45520ee5c980b2133d